README.md

```diff
@@ -1 +1,212 @@
-# Doctor-Reservation-backend
+<a name="readme-top"></a>
```

Comment:
# Frontend re-review

## The mobile view still needs some work to be serviceable:

- [ ] The elements are not centered in the mobile view, and the sidebar sometimes appears behind the content:


![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/ad49f4f3-bafb-4b3d-a9ed-0b5191350464)

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/d3269dc5-e79f-438a-89b8-08f217a75689)

This is because the sidebar container is getting in the way of the other content.

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/d9cb05b3-fd40-4c42-9edb-e0502218c0f6)

Also a lot of padding:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/628dfa01-8f95-431c-8b01-40e59ffb356c)

The way to solve this problem is to use `position: absolute` in order to position the elements so they don't interfere with each other:

Also with `z-index:200` we ensure the sidebar is the topmost element and it is never behind the content.

In order to solve these issues I'll give you an example (this is not an official solution, just an example with ideas/solutions to play around with in order for you to discover what is the best route):

```css
.sidebar_container_items,  .nav_btn {
position:absolute
}

.sidebar_container {
position:absolute
z-index: 200

{
```

You can try it in the browser dev-tools in the `Elements` tab first if you want to see if it works.

This way, the navbar container won't get in the way, and the rest of the content is centered:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/19bd8a80-7034-4f65-bb7b-395bd9edf460)

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/8c79c2c7-76e9-4d29-af54-977ff4f5a3f8)

That way, the mobile view will be more functional. 👍 


- [ ] The navbar sometimes can't be opened or closed.

This is because it sometimes gets trapped within the rest of the contents containers (in this case, the doctor's containers traps the "X" and it's no longer clickable).

The solution I wrote earlier helps a lot with this.  Play with the `z-index` property to send elements to the background or the foreground.

_Note: In order to `z-index` to work, the element must be positioned with `position: absolute` or `position: relative`._

Now that you have some solutions/ideas, kindly fix this navbar functionality. You can do it. 💪

Reply:
@Meltrust 
thanks for your good and valuable feedback according to review below changes are brought